src/views/wishlist/ui/WishlistPage.tsx

```
@@ -19,6 +19,8 @@ export default function WishListPage() {
 
   const { data: wishlist, isLoading } = useWishlist();
 
+  const { data: wishlist, isLoading } = useWishlist();
```

Biome (2.4.4)

[error] Shouldn't redeclare 'wishlist'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)